dafni_cli/datasets/dataset.py

```diff
@@ -61,19 +61,26 @@ class Dataset(ParserBaseObject):
         ParserParam("date_range_end", ["date_range", "end"], parse_datetime),
     ]
 
-    def output_dataset_details(self):
-        """Prints relevant dataset attributes to command line"""
-        click.echo("Title: " + self.title)
-        click.echo("ID: " + self.dataset_id)
-        click.echo("Latest Version: " + self.version_id)
-        click.echo("Publisher: " + self.source)
+    def output_brief_details(self):
+        """Prints this datasets brief details e.g. for the get datasets command"""
+        click.echo("-" * CONSOLE_WIDTH)
+        click.echo(self.title)
+        click.echo("")
+        click.echo(
+            f"ID: {self.dataset_id}{TAB_SPACE}Latest version ID: {self.version_id}"
+        )
         click.echo(
+            f"Publisher: {self.source}{TAB_SPACE}"
             f"From: {format_datetime(self.date_range_start, include_time=False)}{TAB_SPACE}"
             f"To: {format_datetime(self.date_range_end, include_time=False)}"
         )
-        click.echo("Description: ")
+        click.echo("")
         prose_print(self.description or "", CONSOLE_WIDTH)
         click.echo("")
+        click.echo(
```

Comment:
I know this aligns with the front-end but I think it has the potential to be confusing - it either needs to be more obvious that it's the file types or removed - I'm leaning more towards removed